package.json

```diff
@@ -59,7 +59,7 @@
     "prettier": "^2.0.5",
     "react-test-renderer": "^16.13.1",
     "sinon": "^9.0.2",
-    "typescript": "^3.9.7",
+    "typescript": "^4.0.3",
```

Comment:
It shouldn't hurt, but were we expecting these changes to come along in the update?